networking_oneview/ml2/drivers/oneview/mech_oneview.py

```diff
@@ -16,41 +16,36 @@
 from oslo_log import log
 try:
     from neutron_lib.api.definitions import portbindings
-except ImportError:
-    from neutron.extensions import portbindings
-try:
     from neutron_lib.plugins import constants as p_const
-except ImportError:
-    from neutron.plugins.common import constants as p_const
-try:
     from neutron_lib.plugins.ml2 import api
 except ImportError:
+    from neutron.extensions import portbindings
+    from neutron.plugins.common import constants as p_const
     from neutron.plugins.ml2 import driver_api as api
 
 from networking_oneview.conf import CONF
 from networking_oneview.ml2.drivers.oneview import common
-from networking_oneview.ml2.drivers.oneview.neutron_oneview_client import (
-    Client)
+from networking_oneview.ml2.drivers.oneview import neutron_oneview_client
 from networking_oneview.ml2.drivers.oneview import synchronization
 
 LOG = log.getLogger(__name__)
 
 
 class OneViewDriver(api.MechanismDriver):
-    def initialize(self):
+    def __init__(self):
```

Comment:
Would not it be overhead if we set all those values when the Object is created instead of when it is initialized?